.gitlab-ci.yml

```diff
@@ -740,3 +740,72 @@ tox-inventory-builder:
     - cd contrib/inventory_builder && tox
   when: manual
   except: ['triggers', 'master']
+
+
+# Tests for contrib/terraform/
+.terraform_install: &terraform_install
+  <<: *job
+  before_script:
+    # Set Ansible config
+    - cp ansible.cfg ~/.ansible.cfg
```

Comment:
why you copy to ~/.ansible if you run ansible into same directory, it's not needed: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-configuration-settings-locations

Reply:
@Atoms I wasn't sure about this, so I did a test run: https://gitlab.com/kargo-ci/kubernetes-sigs-kubespray/-/jobs/160168839

This is needed because there is a `cd inventory/$CLUSTER` later, so the current directory changes and there is no `inventory/$CLUSTER/ansible.cfg` there. Do you have another suggestion?

Reply:
ah then it's ok, wasn't sure about this step, if it's needed then needed np, it's not time consuming operation :)